src/app/editor-common/components/key-value-component/key-value-input.component.ts

```diff
@@ -0,0 +1,129 @@
+import {Component, forwardRef, Input} from "@angular/core";
+import {
+    NG_VALUE_ACCESSOR,
+    ControlValueAccessor,
+    FormGroup,
+    FormBuilder,
+    NG_VALIDATORS,
+    FormControl
+} from "@angular/forms";
+import {ExpressionModel} from "cwlts/models/d2sb";
+import {ComponentBase} from "../../../components/common/component-base";
+
+require("./key-value-input.component.scss");
+
+@Component({
+    selector: "[ct-key-value-input]",
```

Comment:
The reason I made it and attribute selector was that I was having issues with the css styles. I will try to find a better solution.